src/View/Admin/edit.html.twig

```diff
@@ -77,41 +77,56 @@
                         </div>
                     </div>
 
-                    <div class="form-group row mt-3">
-                        <label for="price" class="label_form_add_BD col-sm-2 control-label">PRIX</label>
-                        <div class="col-sm-8">
-                            <input type="text" class="form-control input-bg" name="price" id="price" value="{{ comicBook.price }}">
+                    {% set index = 0 %}
+                    {% for author in comicAuthors %}
+                        {% if comicBook.id ==  author.comic_book_id %}
+                        <div class="form-group row mt-3">
+                            <label for="author_id{{ index }}" class="label_form_add_BD col-sm-2 control-label ">AUTEUR <span class="asterisk">*</span></label>
+                                <div class="col-sm-8">
+                                    <select class="form-select input-bg" name="author_id{{ index }}" id="author_select" aria-label="Example select with button addon" required>
```

Comment:
If you put an aria-label, which is very good by the way, please take care of what you put inside => "Example select with button addon" is not a pertinent text ...

Reply:
I changed the text in the aria-label. I think its good for now.